examples/canard/Kconfig

```diff
@@ -6,7 +6,7 @@
 config EXAMPLES_LIBCANARD
 	tristate "libcandard example"
 	default n
-	depends on CANUTILS_LIBCANARD && CLOCK_MONOTONIC && SYSTEM_TIME64
+	depends on CANUTILS_LIBCANARD && SYSTEM_TIME64
```

Comment:
There was no change to remove CLOCK_MONOTONIC from canard_main.c, so this breaks the build.

Reply:
ditto